nilearn/_utils/validation.py

```diff
@@ -0,0 +1,153 @@
+"""
+Utilities to check for valid instances
+"""
+import warnings
+
+from sklearn.base import clone
+from sklearn.externals.joblib import Memory
+from sklearn.feature_selection import (SelectPercentile, f_regression,
+                                       f_classif)
+
+from .param_validation import _adjust_screening_percentile
+from .compat import _basestring
+from ..input_data import NiftiMasker, MultiNiftiMasker
+
+
+def check_masking(mask, target_affine=None, target_shape=None,
```

Comment:
Naming is inconsistent with `check_niimg` IMHO. This does not simply load the mask and do sanity checks.